### PR TITLE
readme update, version bump, mod update

### PR DESCRIPTION
**Pull request by jeevatkm**
For release v2.3.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,7 @@ fmt.Println("Response Info:")
 fmt.Println("Error      :", err)
 fmt.Println("Status Code:", resp.StatusCode())
 fmt.Println("Status     :", resp.Status())
+fmt.Println("Proro      :", resp.Proto())
```
Comment (moorereason) [inline README.md:134]: s/Proro/Proto/

Comment (jeevatkm) [reply README.md]: Corrected.

File: README.md [hunk 2]
```diff
@@ -154,27 +156,29 @@ Response Info:
 Error      : <nil>
 Status Code: 200
 Status     : 200 OK
-Time       : 465.301137ms
-Received At: 2019-06-16 01:52:33.772456 -0800 PST m=+0.466672260
+Proro      : HTTP/2.0
```
Comment (lrita) [inline README.md:159]: s/Proro/Proto/g

Comment (jeevatkm) [reply README.md:159]: Again. I missed one more spot 🤦

Comment (jeevatkm) [reply README.md]: Corrected.

**Pull request merged**